db/migrate/20221011192713_add_targets_count_to_users.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddTargetsCountToUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :targets_count, :integer, default: 0, null: false
```

Comment:
Why do we need a new attribute for this? Can we just call `user.targets.count` to get this "on the fly" instead of persist this number?

Reply:
Just saw @sebastiancaraballo comment. I think it's ok to keep it as it is right now if you want 🙂 not sure about the real benefits of this though

Reply:
Yeah I saw that later 😅. I think it's ok to keep it. I have some opinions against this, but that's actually not relevant here